tests/quantization/quanto_integration/test_quanto.py

```diff
@@ -223,7 +223,9 @@ def test_serialization_bin(self):
         with tempfile.TemporaryDirectory() as tmpdirname:
             with self.assertRaises(ValueError) as e:
                 self.quantized_model.save_pretrained(tmpdirname, safe_serialization=False)
-            self.assertIn("The model is quantized with quanto and is not serializable", str(e.exception))
+            self.assertIn(
+                "The model is quantized with QuantizationMethod.QUANTO and is not serializable", str(e.exception)
+            )
```

Comment:
`modeling_utils.py` changed log, as in https://github.com/huggingface/transformers/blob/a52478253bbe522a420e88ea3940d4d98a935300/src/transformers/modeling_utils.py#L3633, change here accordingly to make test pass.